models/product/src/product-variant/product-variant-draft/presets/sample-data-goodstore/bedding-bundle-01.ts

```diff
@@ -0,0 +1,14 @@
+import { AttributeDraft } from '../../../../attribute';
+import type { TProductVariantDraftBuilder } from '../../../types';
+import * as ProductVariantDraft from '../../index';
+
+const beddingBundle01 = (): TProductVariantDraftBuilder =>
+  ProductVariantDraft.presets
+    .empty()
+    .sku('DCC-03')
+    .key('bed-bundle-1')
+    .prices([])
+    .images([])
+    .attributes([AttributeDraft.random().name('product-ref').value(null)]);
```

Comment:
The value is `null`?

Reply:
Yeah 🤷 It wouldn't accept `undefined`, had to go with `null`

Reply:
Your raise a good point - since this is stringified, might be best to go with empty string instead

Reply:
Fixed in e83c2e426ae65b940293d55bd1d5fe5a8a693093